assets/src/web-stories-block/block/block-types/latest-stories/edit.js

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { useDebouncedCallback } from 'use-debounce';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * LatestStoriesEdit props.
+ *
+ * @typedef LatestStoriesEditProps
+ *
+ * @property {Object}   attributes    Block attributes.
+ * @property {()=>void} setAttributes Callable function for saving attribute values.
+ */
+
+/**
+ * Internal dependencies
+ */
+import StoriesInspectorControls from '../../components/storiesInspectorControls';
+import StoriesBlockControls from '../../components/storiesBlockControls';
+import StoriesLoading from '../../components/storiesLoading';
+import { FETCH_STORIES_DEBOUNCE } from '../../constants';
+import StoriesPreview from '../../components/storiesPreview';
+
+/**
+ * Module constants
+ */
+const LATEST_STORIES_QUERY = {
+  per_page: 20,
+  _embed: 'author',
+};
+
+/**
+ * LatestStoriesEdit component
+ *
+ * @param {LatestStoriesEditProps} props Component props.
+ *
+ * @return {*} JSX markup for the editor.
+ */
+const LatestStoriesEdit = ({ attributes, setAttributes }) => {
+  const {
+    blockType,
+    align,
+    viewType,
+    numOfStories,
+    numOfColumns,
+    orderByValue,
+    viewAllLinkLabel,
+    authors,
+    isStyleSquared,
+  } = attributes;
+
+  const [fetchedStories, setFetchedStories] = useState([]);
+  const [isFetchingStories, setIsFetchingStories] = useState([]);
+
+  /**
+   * Fetch stories based on the query.
+   *
+   * @return {void}
+   */
+  const fetchStories = async () => {
+    try {
+      setIsFetchingStories(true);
+      const stories = await apiFetch({
+        path: addQueryArgs('/web-stories/v1/web-story', LATEST_STORIES_QUERY),
```

Comment:
Can use the API path from `webStoriesBlockSettings` instead of hardcoding `/web-stories/v1/web-story`

Reply:
You can just add that [here](https://github.com/google/web-stories-wp/blob/dd2783fc1585d63e5c196ffdbb3b4fcd60684530/includes/Block/Embed_Block.php#L112-L116).